package.json

```diff
@@ -178,6 +178,7 @@
     "react-intl": "2.7.2",
     "react-lottie": "1.2.3",
     "react-markdown": "3.1.0",
+    "react-middle-truncate": "^1.0.3",
```

Comment:
@daniloprates Would you please update this to use an exact version rather than `^#.#.#`?
We are using specific versions of packages in our app as you know.